ts/features/wallet/component/NewMethodAddedNotifier.tsx

```diff
@@ -51,7 +24,9 @@ const NewPaymentMethodAddedNotifier = (props: Props) => {
     string
   >("");
 
-  const { present } = newBancomatBottomSheet(props.startCoBadgeOnboarding);
+  const { present } = BancomatInformationBottomSheet(
```

Comment:
```suggestion
  const { present } = bancomatInformationBottomSheet(
```
To keep consistency in the codebase, we use `PascalCase` for components and types, `camelCase` for functions and const.